src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -782,9 +846,31 @@ public void deleteTicketField(long id) {
     complete(submit(req("DELETE", tmpl("/ticket_fields/{id}.json").set("id", id)), handleStatus()));
   }
 
+  public Iterable<SuspendedTicket> getSuspendedTickets(Map<String, List<String>> parameters) {
+    return new PagedIterable<>(
+        cbp("/suspended_tickets.json", map2ParamList(parameters)),
+        handleList(SuspendedTicket.class, "suspended_tickets"));
+  }
+
   public Iterable<SuspendedTicket> getSuspendedTickets() {
     return new PagedIterable<>(
-        cnst("/suspended_tickets.json"), handleList(SuspendedTicket.class, "suspended_tickets"));
+        cbp("/suspended_tickets.json"), handleList(SuspendedTicket.class, "suspended_tickets"));
+  }
+
+  public Iterable<Activity> getActivities(Map<String, List<String>> parameters) {
```

Comment:
Looks like a new endpoint support, good idea, but could we move it into its own PR?